app/models/workflow_processor.rb

```diff
@@ -136,26 +137,18 @@ def perform_math(action, payload)
   end
 
   def model_update(step, payload)
-
-    puts "model_update"
-    puts step.inspect
-    puts payload.inspect
-
-
-
-
     resource_type = payload[:resource_type]
     resource = JSON.parse(payload[:resource])
-
-    puts resource.inspect
-    puts resource_type.inspect
-
     
     field = step["field"]
     value = get_value(resource, step["value"])
     record = resource_type.constantize.find(resource["id"])
-
-
+    record.update!(field => value)
```

Comment:
Why you added this here..Logic is already inside ModelUpdate Handler...Check this file: update_model_handler.rb